src/test/ui/issues/issue-52717.stderr

```diff
@@ -2,10 +2,16 @@ error[E0026]: variant `A::A` does not have a field named `fob`
   --> $DIR/issue-52717.rs:10:12
    |
 LL |     A::A { fob } => { println!("{}", fob); }
-   |            ^^^
-   |            |
-   |            variant `A::A` does not have this field
-   |            help: a field with a similar name exists: `foo`
+   |            ^^^ variant `A::A` does not have this field
+   |
+help: `A::A` has a field named `foo`, try replacing `fob` for it.
+   |
+LL |     A::A { foo } => { println!("{}", fob); }
+   |            ~~~
+help: a field with a similar name exists
+   |
+LL |     A::A { foo } => { println!("{}", fob); }
+   |            ~~~
```

Comment:
I'm not sure we need both of these suggestions to appear in the error message. I think the second one is maybe the most accurate (?) but I could see a case for either of them to be kept.